src/App.js

```diff
@@ -16,22 +16,22 @@ class App extends Component {
   updateSearchValue = (value) => this.setState({searchValue: value});
 
   handleSubmit = async () => {
-    const url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATION_KEY}&q=${this.state.searchValue}&format=json`
-
-    let response = await axios.get(url);
-    this.setState({cityData: response.data[0]});
+    const cityUrl = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATION_KEY}&q=${this.state.searchValue}&format=json`;
+    let cityResponse = await axios.get(cityUrl);
+    cityResponse.data[0].mapUrl = `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATION_KEY}&center=${cityResponse.data[0].lat},${cityResponse.data[0].lon}&zoom=12`;
+    this.setState({cityData: cityResponse.data[0]});
```

Comment:
I like this approach of storing map data inside of your existing location data. One less thing to pass as props, which helps keep the code clean!